doc/release/RELEASE-NOTES.md

```diff
@@ -49,8 +50,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Database changes
 * *Describe high-level database changes.*
 
-#### Migrations:
-* 0170_add_uuid_to_relevant_tables.py - A new `uuid` field was added to the following tables:
+#### Migrations
+* 0170_add_uuid_to_relevant_tables - A new `uuid` field was added to the following tables:
```

Comment:
Do we need to remove .py?

Reply:
Otherwise, it feels like the file name is incomplete without the .py extension. Using dashes in the name makes it seem like something is missing. Or why use dashes instead of proper spaces between words?